### ISSUE TITLE
With concatenated df, unable to do df.head()

### ISSUE THREAD

**Issue by cyril-p** [vaexio/vaex#563]
```
filenames = glob.glob(folder + "/*.hdf5")
df = vaex.open_many(filenames)
df.head()`
```

error:

`
ValueError: array is of length 11, while the length of the DataFrame is 10
`

However, `df.head` works

**Comment by JovanVeljanoski** [comment]
Hi!

Thanks for reporting this. This is a know issue and work has started #531.

In the meantime, if possible you can export the concatenated dataframe and use the single export. 

Cheer!

**Comment by maartenbreddels** [comment]
Should be fixed by #590

**Comment by maartenbreddels** [comment]
Fixed indeed, tested by #531
